postcss.config.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable prefer-import/prefer-import-over-require */
```

Comment:
As said in the comment of `babel.config.js`, I think this configuration should look to the same target as that one.

Also, we should inlining any fonts used to avoid flashes of layout.